src/app/lib/syndesis-data-mapper/services/error.handler.service.ts

```diff
@@ -27,36 +27,30 @@ export class ErrorHandlerService {
     public info(message: string, error: any) { this.addError(message, ErrorLevel.INFO, error); }
     public warn(message: string, error: any) { this.addError(message, ErrorLevel.WARN, error); }
     public error(message: string, error: any) { this.addError(message, ErrorLevel.ERROR, error); }
-    public validationError(message: string, error: any) { this.addError(message, ErrorLevel.VALIDATION_ERROR, error); }
+    public validationError(message: string, error: any) { this.addValidationError(message, ErrorLevel.VALIDATION_ERROR, error); }
```

Comment:
Not really in the code, but why using both, a method name 'addValidationError' and an `ErrorLevel.VALIDATION_ERROR`. Feels redundant to me (e.g. when the method is only about validation errors, then you don't need the second argument as you can set the level within the method itself)

Reply:
Fair point. I'll change that.